proposals/harbor.adoc

```diff
@@ -0,0 +1,134 @@
+
+ * Trend Micro
+ * OnStar in China
+ * Caicloud
+ * CloudChef
+ * Rancher
+
+A number of CNCF member companies, such as JD.com, China Mobile, Caicloud, Dataman, and Tenxcloud are also users of Harbor.
+
+== In-Flight Features
+
+The Harbor team is currently working on improving Harbor, including:
+
+ * Native support of Helm
+ * Highly-available deployments
+ * Image caching and proxying
+ * Label-related feature improvements
+ * Quotas
+
+The direction of the project has been generally guided by our open source community and users. There are a plethora of GitHub issues requesting various features that we prioritize based on popularity of user requests and engineering capacity. Our community has been involved in the addition of several new important features, including the creation of a Helm chart for Harbor.
+
+A roadmap for future features, including those listed above, can be found GitHub: https://github.com/vmware/harbor/labels/Epic. The project welcomes contributions of any kind: code, documentation, bug reporting via issues, and project management to help track and prioritize workstreams.
+
+== Use Cases
+
+The following is a list of common use-cases for Harbor users:
+
+ * *On-prem container registry* – organizations with the desire to host sensitive production images on-premises can do so with Harbor
+ * *Vulnerability scanning* – organizations can scan images before they are used in production. Images with failed vulnerability scans can be blocked from being pulled
+ * *Image signing* – images can be signed via Notary to ensure provenance
+ * *Role-based Access Control* – integration with LDAP (and AD) to provide user- and group-level permissions
+ * *Image replication* – production images can be replicated to disparate Harbor nodes, providing disaster recovery, load balancing and the ability for organizations to replicate images to different geos to provide a more expedient image pull
+
+
+== CNCF Donation Details
+ * *Preferred Maturity Level:* Sandbox or Incubation
+ * *Sponsors:* Quinton Hoole and Ken Owens
+ * *License:* Apache 2
+ * *Source control repositories / issue tracker:* https://github.com/vmware/harbor, with a ZenHub board tracking engineering work. _Will be moved to github.com/goharbor organization_
+ * *Infrastructure Required:* Infrastructure for CI / CD
+ * *Website:* https://vmware.github.io/harbor/. Will be moved to https://goharbor.io.
+ * *Release Methodology and Mechanics:* We currently do feature releases for major updates 3-4 times per year (with minor releases) when needed. Before releasing we tag one or more RC releases for community testing. Commits to the project are analyzed and we require that changes do not decrease overall test coverage to the project.
+
+== Social Media Accounts:
+
+ * *Twitter:* https://twitter.com/project_harbor
+ * *Users Google Groups:* harbor-users@googlegroups.com
+ * *Developer Google Groups:* harbor-dev@googlegroups.com
+ * *Slack:* #harbor on https://code.vmware.com/join/
```

Comment:
this will have to move to the CNCF slack or a neutral slack

Reply:
Yes, absolutely. We're in the process of setting up a new Slack workspace @caniszczyk.